laws/src/main/scala/cats/laws/discipline/Eq.scala

```diff
@@ -12,8 +13,10 @@ object eq {
    * and comparing the application of the two functions.
    */
   implicit def function1Eq[A, B](implicit A: Arbitrary[A], B: Eq[B]): Eq[A => B] = new Eq[A => B] {
+    val sampleCnt: Int = if (Platform.isJvm) 50 else 5
```

Comment:
Would it be better to keep the JVM side at 100? Is there example builds where this has failed for lack of output when building the JVM side?